src/index.html

```diff
@@ -7,13 +7,90 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Dia</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,300..800;1,300..800&family=Poppins:wght@400;500;600;700&family=Roboto&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/main.scss"
     />
   </head>
   <body>
-    <h1>Dia</h1>
+    <header class="header">
+      <nav class="navigation header__navigation">
+        <a
+          href="#logo"
+          class="navigation__item navigation__item--logo"
+        >
+          <img
+            src="./images/logo/dia-logo.svg"
+            alt="logo"
+          />
+        </a>
+
+        <a
+          href="#menu"
+          class="navigation__item navigation__item--menu"
+        >
+          <img
+            src="./images/Vector.svg"
+            alt="menu"
+          />
+        </a>
+
+        <a
+          href="#hire-us"
+          class="navigation__item navigation__item--hire-us"
+        >
+          hire us
+        </a>
+      </nav>
+
+      <div class="header__info-block">
+        <h1 class="header__title">Strategic Agency</h1>
+
+        <p class="header__paragraph">
+          We believe in the power of bold ideas that can solve business
+          challenges.
+        </p>
+
+        <a
+          href="#learn-more"
+          class="button header__button"
+        >
+          Learn more
+        </a>
+      </div>
+
+      <div class="header__slider-block">
+        <img
+          src="../src/images/slider/slide-img-1.jpg"
+          alt="slide"
+          class="header__slider-image"
+        />
+      </div>
```

Comment:
you should center position of this image and only clip sides (you can change it for the one used in design to better see differences)